apps/nest-server/src/app/app.controller.ts

```diff
@@ -14,4 +14,13 @@ export class AppController {
       console.error(err)
     }
   }
+
+  @Get('presto-error')
+  async getDataWithError() {
+    try {
+      return await this.appService.getDataWithError()
+    } catch (err) {
+      console.error(err)
```

Comment:
I guess this would be an error other than PrestoError because the PrestoError is handled in the `getDataWithError` and no rethrow.

Reply:
yeah, I guess I'm using the service file as the showcase for the error
I don't know if we should also do something else here in the controller